src/content/learn/editor-setup.md

```diff
@@ -49,14 +49,14 @@ title: 에디터 설정하기
 3. `ext install esbenp.prettier-vscode`라고 입력하기
 4. 엔터 누르기
 
-#### 저장 시점에 포맷팅하기 {/*formatting-on-save*/}
+#### 저장 시점에 포매팅하기 {/*formatting-on-save*/}
 
-저장할 때마다 코드가 포맷팅되는 것이 가장 이상적일 것입니다. 이러한 설정은 VS Code에 자체적으로 내장되어있습니다!
+저장할 때마다 코드가 포매팅 되는 것이 가장 이상적일 것입니다. 이러한 설정은 VS Code에 자체적으로 내장되어 있습니다!
 
 1. VS Code에서 `CTRL/CMD + SHIFT + P` 누르기
 2. "settings"라고 입력하기
 3. 엔터 누르기
 4. 검색 창에서 "format on save"라고 입력하기
 5. "format on save" 옵션이 제대로 체크되었는지 확인하세요!
 
-> Prettier는 때때로 다른 린터들과 충돌을 일으킬 수도 있습니다. 하지만 일반적으로 다른 린터들과 함께 깔끔하게 병행해서 동작하도록 하는 방법이 존재합니다. 예를 들어 Prettier를 ESLint와 함께 사용하는 경우 [eslint-prettier](https://github.com/prettier/eslint-plugin-prettier) 플러그인을 설치하면 ESLint 규칙으로서 Prettier가 동작하도록 설정할 수 있습니다.
\ No newline at end of file
+> 만약 ESLint 프리셋에 포매팅 규칙이 있는 경우 Prettier와 충돌을 일으킬 수도 있습니다. ESLint가 *오직* 논리적 실수를 잡는 데만 사용되도록 [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier)를 사용하여 ESLint 프리셋의 모든 포매팅 규칙을 비활성화하는 것을 권장합니다. Pull request가 merge 되기 전에 파일 형식이 지정되도록 하려면 지속적인 통합을 위해 [`prettier --check`](https://prettier.io/docs/en/cli.html#--check)를 사용하세요.
```

Comment:
원문과 다르게 번역되어 있어 수정한 부분입니다.
- 원문
> If your ESLint preset has formatting rules, they may conflict with Prettier. We recommend disabling all formatting rules in your ESLint preset using [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) so that ESLint is *only* used for catching logical mistakes. If you want to enforce that files are formatted before a pull request is merged, use [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) for your continuous integration.

cf. https://react.dev/learn/editor-setup#formatting-on-save